terraform/main.tf

```diff
@@ -1,10 +1,7 @@
 resource "aws_instance" "this" {
-  ami           = data.aws_ami.this.id
+  ami           = "ami-0e4a0478fee8d8aae"
```

Comment:
Currently this is statically configured but the plan is to move away from this and be able to just take the latest AMI we've built when running a training pipeline